reference/5.0/PowershellGet/Unregister-PSRepository.md

```diff
@@ -20,20 +20,25 @@ Unregister-PSRepository [-Name] <String[]> [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-The **Unregister-PSRepository** cmdlet unregisters a repository for the current user.
+
+The `Unregister-PSRepository` cmdlet unregisters a repository for the current user.
 
 ## EXAMPLES
 
 ### Example 1: Unregister a repository
-```
-PS C:\> Unregister-PSRepository -Name "myNuGetSource"
-```
 
-This command unregisters the repository named myNuGetSource.
+This example unregisters the repository named myNuGetSource.
 
-### Example 2: Unregister all repositories
+```powershell
+Unregister-PSRepository -Name "myNuGetSource"
 ```
-PS C:\> Get-PSRepository | Unregister-PSRepository
+
+### Example 2: Unregister all repositories
+
+This example uses `Get-PSRepository` to get all registered repositories, and uses the pipeline operator to pass them to `Unregister-PSRepository` to unregister them.
+
+```powershell
+Get-PSRepository | Unregister-PSRepository
 ```
 
 This command uses Get-PSRepository to get all registered repositories, and uses the pipeline operator to pass them to **Unregister-PSRepository** to unregister them.
```

Comment:
With the description above the example, this line should be removed.

Reply:
You couldn't unregister PSGallery in that version of PowerShellGet. That is why I left it.

Reply:
Wasn’t clear.  The nearly same paragraph is now above and below that sample. Only need one of them.

Reply:
Ooops. Missed cleaning up that paragraph. Fixed now.